src/Style/Sass/abstract/_variables.scss

```diff
@@ -1,5 +1,35 @@
+
 $whitecolor: #e0e0e0;
 $googlecolor: #df4930;
 $twittercolor:#1aa5d8;
 $facebookcolor:#103d80;
 $iconcolor: white;
+$color-primary: black;
+$color-primary-light: rgba(94, 252, 128, 1);
+$color-primary-dark: rgba(0, 150, 33, 1);
+$color-secondary: rgba(255, 255, 255, 1);
+$color-secondary-dark: rgba(204, 204, 204, 1);
+$gray: #ccc;
+$text-primary: rgba(0, 0, 0, 1);
+$white-secondary: rgba(232, 240, 254, 1);
+$white-shade: rgba(232, 240, 254, 0.85);
+$shadow-primary: gray;
+$shadow-primary-dark: rgba(0, 150, 33, 0.3);
+$font-family: Arial;
+$font-size-small: 0.4em;
+$font-size: 1em;
+$font-size-medium: 1.3em;
```

Comment:
It seems like you are not using any of these variables

can you remove them please?